scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -43,17 +43,18 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.14.5/cmake-3.14.5
 FROM base AS libraries
 
 # Boost
-RUN git clone -b boost-1.69.0 https://github.com/boostorg/boost.git \
-    /usr/src/boost; \
-    cd /usr/src/boost; \
-    git submodule update --init --recursive; \
```

Comment:
Especially the recursive submodule clone was taking ages... and it could use a version update anyways. Rationale for 1.73.0: it's what the emscripten build uses.

Reply:
One could also argue for 1.71.0 (which is what the latest ubuntu LTS version focal/20.04 uses). But I don't expect it to matter too much and the emscripten builds are probably the most heavily used ones, so maybe best to keep it in sync with those.